src/editor/svgedit.js

```diff
@@ -3909,37 +3912,38 @@ editor.init = () => {
   */
   const saveDocProperties = function () {
     // set title
-    const newTitle = $('#canvas_title').val();
-    updateTitle(newTitle);
+    const $imgDialog = document.getElementById('se-img-prop');
+    const newTitle = $imgDialog.shadowRoot.querySelector('#canvas_title').value;
     svgCanvas.setDocumentTitle(newTitle);
 
     // update resolution
-    const width = $('#canvas_width'), w = width.val();
-    const height = $('#canvas_height'), h = height.val();
+    const width = $imgDialog.shadowRoot.querySelector('#canvas_width'), w = width.value;
```

Comment:
see comment below but I would like to avoid accessing the shadowRoot of the element. I expect to have a custom event so the saveDocProperties would just read the values in the event detail.